### DISCUSSION TITLE
Contain http_ Sub_ Module module ？

### DISCUSSION THREAD

**Discussion by l-g-t** [tiangolo/nginx-rtmp-docker#90] [Questions] [answered]
The current compiled version of Nginx does not contain http_ Sub_ Module module, which can add support for sub_ The HTTP of the filter instruction_ Sub_ Module module.？

**Comment by tiangolo** [comment, marked as answer]
I'm not sure, this is just plain Nginx with https://github.com/arut/nginx-rtmp-module on top pre-installed.
